src/aggregate_score.rs

```diff
@@ -54,6 +71,38 @@ pub fn sample_aggregate_scores<const N_HAND: usize, R: Rng>(rng: &mut R, num_sam
     scores
 }
 
+pub fn parallel_sample_aggregate_scores<const N_HAND: usize, R: Rng>(
+    _rng: &mut R,
+    num_samples: u32,
+    number_of_threads: u32,
+) -> AggregateScore {
+    let scores = Arc::new(Mutex::new(AggregateScore::default()));
+    let num_samples_perthread = num_samples / number_of_threads;
+    let num_samples_remainder = num_samples % number_of_threads;
+
+    (0..number_of_threads).into_par_iter().for_each(|_x| {
+        let scores_temp = sample_aggregate_scores::<N_HAND, ThreadRng>(
+            &mut rand::thread_rng(),
+            num_samples_perthread,
+        );
+        let scores = Arc::clone(&scores);
+        let mut scores_clone = scores.lock().unwrap();
+        scores_clone.add(scores_temp);
+    });
+
+    if num_samples_remainder > 0 {
+        let scores_temp = sample_aggregate_scores::<N_HAND, ThreadRng>(
+            &mut rand::thread_rng(),
+            num_samples_remainder,
+        );
+        let scores = Arc::clone(&scores);
```

Comment:
I don't think we need to do this here, as we're back in "single threaded" land (all of the other threads have finished by this point in the program). We could probably make that more clear by doing `Arc::into_inner(scores).unwrap().into_inner().unwrap();` right after the parallel block, and then operating directly on the inner data.

Alternatively, there might be a way to just give one of the threads `num_samples_remainder + num_samples_perthread` samples, and drop this block entirely. Not sure of a good way to do that though.